app/src/main/java/com/infomaniak/mail/data/api/ApiRoutes.kt

```diff
@@ -277,6 +277,16 @@ object ApiRoutes {
     }
     //endregion
 
+    //region Snooze
+    fun snooze(mailboxUuid: String): String {
+        return "${mailMailbox(mailboxUuid)}/snoozes"
+    }
+
+    fun snoozeAction(mailboxUuid: String, snoozeUuid: String): String {
```

Comment:
```suggestion
    fun unsnoozeThread(mailboxUuid: String, snoozeUuid: String): String {
```

Reply:
Technically speaking, when only considering the "route" it has nothing to do with unsnoozing. The route truly is what the API calls a "snooze action". This route can be used with a DELETE to unsnooze but it can be used with a PUT to update the snooze date